README.md

```diff
@@ -29,9 +29,14 @@ an [existing implementation in Go](https://github.com/Consensys/zk-evm/).
 brew install openjdk@17
 ```
 
-### Install the relevant CGo compiler for your platform
+### Native Lib Prerequisites
 
-### Install the Go toolchain
+Linux/MacOs
+* Install the relevant CGo compiler for your platform
```

Comment:
```suggestion
* Install the relevant Go compiler for your platform
```

Reply:
No, CGo: one needs a valid `$CC` for CGo to work.

Reply:
@macfarla I am resolving without apply your suggestion since, as @delehef said, the C compiler is required